client/src/redux/slices/auth.slice.js

```diff
@@ -26,6 +27,8 @@ const authSlice = createSlice({
       state.loggedIn = payload.loggedIn
     },
     logout: (state) => {
+      removeUserIDToken()
```

Comment:
Nope, can't call outside functions inside a redux reducer. No side effects. Need to move the removeUserIDToken() and removeUserRefreshToken() somewhere outside this reducer.